waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java

```diff
@@ -153,8 +156,30 @@ private void startWaggleDance(
       int maxWorkerThreads = hiveConf.getIntVar(ConfVars.METASTORESERVERMAXTHREADS);
       boolean tcpKeepAlive = hiveConf.getBoolVar(ConfVars.METASTORE_TCP_KEEP_ALIVE);
       boolean useFramedTransport = hiveConf.getBoolVar(ConfVars.METASTORE_USE_THRIFT_FRAMED_TRANSPORT);
+      boolean useSSL = hiveConf.getBoolVar(ConfVars.HIVE_METASTORE_USE_SSL);
```

Comment:
Is SSL for the metastore something new they added in 2.3.0?

Reply:
this code is new, I suppose it was added recently

Reply:
OK, so what we're doing here is enabling the client to talk to us using SSL and we then in turn talk to the proxied HMS using SSL too?

Reply:
Also, any way we can test any of this? (I understand it might be quite hard...)

Reply:
I see whether is possible to test this somehow

Reply:
You mean the Hive project doesn't have a nice clean test for this that we can copy? ;P

Reply:
strangely not